docs-src/docs/reference/cli.md

```diff
@@ -117,123 +87,76 @@ The added `logConfiguration` section uses the following template:
 
 #### Usage
-<div class="divTable">
-  <div class="divRow">
-    <div class="divCellNoPadding">**l0 environment create** [--size] [--min-count] [--user-data] [--os] [--ami] _environmentName_ </div>
-  </div>
-</div>
+```
+l0 environment create [--size size | --min-count mincount | 
+    --user-data path | --os os | --ami amiID] environmentName
+```
 
 ####Required parameters
-<div class="divTable">
-  <div class="divRow">
-    <div class="divCellNoWrap">_environmentName_</div>
-    <div class="divCell">A name for the environment.</div>
-  </div>
-</div>
+* `environmentName` - A name for the environment.
 
 ####Optional arguments
-<div class="divTable">
-  <div class="divRow">
-    <div class="divCellNoWrap">--size</div>
-    <div class="divCell">The size of the EC2 instances to create in your environment (default: m3.medium).</div>
-  </div>
-    <div class="divRow">
-    <div class="divCellNoWrap">--min-count</div>
-    <div class="divCell">The minimum number of EC2 instances allowed in the environment's autoscaling group (default: 0).</div>
-  </div>
-  <div class="divRow">
-    <div class="divCellNoWrap">--user-data</div>
-    <div class="divCell">The user data template to use for the environment's autoscaling group.</div>
-  </div>
-  <div class="divRow">
-    <div class="divCellNoWrap">--os</div>
-    <div class="divCell">The operating system used in the environment. Options are "linux" or "windows" (default: linux).
-        More information on windows environments is documented below</div>
-  </div>
-  <div class="divRow">
-    <div class="divCellNoWrap">--ami</div>
-    <div class="divCell">A custom AMI ID to use in the environment. If not specified, Layer0 will use its default AMI ID for the specified operating system.</div>
-  </div>
-</div>
-
-The user data template can be used to add custom configuration to your Layer0 environment.
+* `--size size` - The instance size of the EC2 instances to create in your environment (default: m3.medium).
+* `--min-count mincount` - The minimum number of EC2 instances allowed in the environment's autoscaling group (default: 0).
+* `--user-data path` - The user data template file to use for the environment's autoscaling group.
+* `--os os` - The operating system used in the environment. Options are "linux" or "windows" (default: linux). More information on windows environments is documented below.
+* `ami amiID` - A custom EC2 AMI ID to use in the environment. If not specified, Layer0 will use its default AMI ID for the specified operating system.
+
+The user data template can be used to add custom configuration to your Layer0 environment. They are usually scripts that are executed at instance launch time to ensure an EC2 instance is in the correct state after the provisioning process finishes.
 Layer0 uses [Go Templates](https://golang.org/pkg/text/template) to render user data.
 Currently, two variables are passed into the template: **ECSEnvironmentID** and **S3Bucket**.
```

Comment:
You need to update the windows default template below, only one line change:
* From `$agentVersion = 'v1.14.0-1.windows.1'`
* To `$agentVersion = 'v1.5.2'`

Reply:
:white_check_mark: